concrete/blocks/express_entry_list/controller.php

```diff
@@ -258,8 +312,7 @@ public function save($data)
 
         $entity = $this->entityManager->find('Concrete\Core\Entity\Express\Entity', $data['exEntityID']);
         if (is_object($entity) && is_array($this->request->request->get('column'))) {
-
-            $provider = $this->app->make('Concrete\Core\Express\Search\SearchProvider', array($entity, $entity->getAttributeKeyCategory()));
+            $provider = $this->app->make('Concrete\Core\Express\Search\SearchProvider', [$entity, $entity->getAttributeKeyCategory()]);
```

Comment:
Maybe we can use `$this->app->make(\Concrete\Core\Express\Search\SearchProvider::class)` instead? Or import the class dependency?

Reply:
I can follow the coding style guideline but I couldn't find which is the best way